File: Perez-Meneses-Juan-Carlos/Procesos/process3.m
#import <Foundation/Foundation.h>
#import <unistd.h>
#import <stdlib.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        NSLog(@"argc = %d", argc - 1);
        
        if (argc == 1) {
            return 0;
        } else {
            return argc;
        }
    }
}

File: Perez-Meneses-Juan-Carlos/Procesos/process3.c
#include<stdio.h>
#include<unistd.h>
#include<time.h>
#include<stdlib.h>


int main(int argc, char **argv)
{ 
	printf("argc = %d \n", argc-1);
	if (argc == 1)
		return 0;
	else
		return argc;
}


